lib/SPIRV/libSPIRV/SPIRVEnum.h

```diff
@@ -135,6 +136,7 @@ template <> inline void SPIRVMap<SPIRVExtInstSetKind, std::string>::init() {
       "NonSemantic.Shader.DebugInfo.100");
   add(SPIRVEIS_NonSemantic_Shader_DebugInfo_200,
       "NonSemantic.Shader.DebugInfo.200");
+  add(SPIRVEIS_Aux_Data, "NonSemantic.AuxData");
```

Comment:
Since this is a NonSemantic EIS:
```suggestion
  add(SPIRVEIS_NonSemantic_AuxData, "NonSemantic.AuxData");
```